Description: Correct pod NAME issue in LedgerSMB/PE.pm
 Has an incomplete module name and does not have the dash plus short description
 in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472612&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/PE.pm
+++ b/LedgerSMB/PE.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-PE
+LedgerSMB::PE - Support functions for projects, partsgroups, and parts.
 
 =head1 SYNOPSIS
 
